.github/workflows/alpine_x86_64_release.yml

```diff
@@ -0,0 +1,43 @@
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Is this mean to run on `ubuntu-latest` for alpine? The `gnu` script looks like the same action.

Reply:
This is expected to be run in official alpine docker image.

Check this:

https://github.com/zw963/wordsmith/blob/main/.github/workflows/alpine_x86_64_release.yml#L9-L10

In fact, i consider the gnu_x86_64_release is not so necessary., i add it just because i want to ensure it works,
and i change apple darwin release workflow from it.